modules/server_new/src/main/scala/observe/server/ObserveEngine.scala

```diff
@@ -91,6 +92,15 @@ trait ObserveEngine[F[_]] {
     runOverride: RunOverride
   ): F[Unit]
 
+  def nextAtom(
```

Comment:
May I suggest adding a verb since this seems to be an action method? For example, `loadNextAtom`